apis/cache/v1alpha1/types.go

```diff
@@ -37,6 +37,7 @@ type RedisClusterSpec struct {
 // to a Redis managed resource such as a GCP CloudMemorystore instance or an AWS
 // ReplicationGroup. Despite the name RedisCluster claims may bind to Redis
 // managed resources that are a single node, or not in cluster mode.
+// +kubebuilder:resource:categories={crossplane,claim}
```

Comment:
does it make sense to add our types to the `all` category too? (maybe not ALL of them, but the important/useful ones?)

Reply:
I think that could make good sense! I opted not to here because I thought that could be a more controversial change. Which ones would you consider worthy of being added to all? Given that any project that installs CRDs could add their’s to the all category, I would lean towards being conservative with adding it :)

Reply:
Agreed that it would be a more controversial change.  Looking through the list of CRDs, i'm not really seeing any that clearly stick out as being important enough to be at the `all` level on par with pods and services.  Let's leave any `all` changes out of this PR :)